snippets/man_gen/hubl_general_variables.json

```diff
@@ -1,77 +1,142 @@
 {
```

Comment:
Were `account`, `contact`, `content`, and `site_settings` omitted intentionally? At first I figured you were leaving out dicts but I see `request_contact` is in there still.